pandas/core/frame.py

```diff
@@ -778,14 +778,55 @@ def style(self):
         return Styler(self)
 
     def iteritems(self):
-        """
+        r"""
         Iterator over (column name, Series) pairs.
 
-        See also
+        Iterates over the DataFrame columns, returning a tuple with the column name and the content as a Series.
```

Comment:
this doesn't fit within 80 chars line length limit (PEP8), I assume the CI is failing due to that.

There are also some other lines below that are too long. I would recommend to activate a flake8 / linter plugin in your editor, or run `flake8` locally. You can also check the output of the failing build on Travis.